app/scripts/views/base.js

```diff
@@ -389,6 +389,29 @@ define(function (require, exports, module) {
 
     // called after the view is visible.
     afterVisible: function () {
+      // restyle side-by-side links to stack if they are too long
+      // to fit on one line
+      var linkContainer = this.$el.find('.links');
+      if (linkContainer.length > 0) {
+        var containerWidth = linkContainer.width();
+        var halfContainerWidth = containerWidth / 2;
+        var shouldResetLinkSize = false;
+
+        linkContainer.children().each(function (i, item) {
+          var linkWidth = $(item).width();
```

Comment:
also, this new behaviour breaks the settings/change_password view 
![](http://v14d.com/i/5756e2e4a0565.jpg) I'm guessing if we only have link then we should not center it?